server/index.js

```diff
@@ -1,10 +1,25 @@
 require('dotenv/config');
 const express = require('express');
+const fetch = require('node-fetch');
 const staticMiddleware = require('./static-middleware');
+const ClientError = require('./client-error'); // eslint-disable-line
+const errorMiddleware = require('./error-middleware');
 
 const app = express();
 
 app.use(staticMiddleware);
+app.use(express.json());
+
+app.get('/api/parks', (req, res, next) => {
+  fetch('https://developer.nps.gov/api/v1/parks?limit=500&api_key=iIb41q7SpeNVydFcxXQZtBqE2IuRe9WTcy4WGqJz', { headers: { "User-Agent": "Chaim" }} )
```

Comment:
You api key should not be checked into Git as it is private information! This is what your `.env` file is for 😄